<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windows_reg</class>
 <widget class="QMainWindow" name="windows_reg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Регистрация пользователя</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);
font: 600 12pt &quot;Segoe UI&quot;;
color: #fff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Логин*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="user_login">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(45, 45, 45);
border-style: solid;
border-width: 1px;
border-color: #ffffff;
border-radius: 5px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Пароль*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="user_password">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(45, 45, 45);
border-style: solid;
border-width: 1px;
border-color: #ffffff;
border-radius: 5px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Email*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="user_email">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(45, 45, 45);
border-style: solid;
border-width: 1px;
border-color: #ffffff;
border-radius: 5px;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="exit_reg">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(45, 45, 45);
border-style: solid;
border-width: 1px;
border-color: #ffffff;
border-radius: 5px;</string>
           </property>
           <property name="text">
            <string>Выйти</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reg">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(45, 45, 45);
border-style: solid;
border-width: 1px;
border-color: #ffffff;
border-radius: 5px;</string>
           </property>
           <property name="text">
            <string>Регистрация</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 600 12pt &quot;Segoe UI&quot;;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
